### DISCUSSION TITLE
Welcome to PySolid Discussions!

### DISCUSSION THREAD

**Discussion by yunjunz** [insarlab/PySolid#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glPeng-seis** [comment]
Hi, Yunjun. I found pysolid only can calculate displacement of three components. Could you update this software to calculate stress/strain tensor and volume strain at a certain position (latitude, longitude and depth).

**Reply by yunjunz** [reply]
Hi @glPeng-seis, we welcome contributions.
